corehq/apps/auditcare/utils/migration.py

```diff
@@ -0,0 +1,61 @@
+from corehq.apps.auditcare.models import AuditcareMigrationMeta
+from django.core.cache import cache
+import gevent
+from dimagi.utils.dates import force_to_datetime
+
+
+class AuditCareMigrationUtil():
+
+    def __init__(self):
+        self.start_key = "auditcare_migration_2021_next_batch_time"
+        self.start_lock_key = f"{self.start_key}_lock"
+
+    def get_next_batch_start(self, counter=0):
+        if self.is_start_key_lock_acquired():
+            gevent.sleep(1)
```

Comment:
Why sleep?

Reply:
I wanted it to have enough time to check back again.